internal/connect/command_offset_status_describe.go.go

```diff
@@ -18,19 +18,19 @@ type alterStatusOut struct {
 
 func (c *offsetCommand) newStatusCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:               "status",
-		Short:             "Get connector offset alter status",
-		Args:              cobra.ExactArgs(1),
+		Use:               "status describe",
```

Comment:
`status` needs to be defined in a separate file. Please take a look at other commands for examples of how to make subcommands.

Reply:
Done